flat_mutation_reader.cc

```diff
@@ -32,6 +32,7 @@
 #include <seastar/util/defer.hh>
 #include "utils/exceptions.hh"
 #include <seastar/core/on_internal_error.hh>
```

Comment:
Changelog should explain why you're doing all this new stuff (and not "for CALL").

Reply:
not part of this PR, explained below